CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-## 2.0.0 (TBD)
+## 2.1.0 (TBD)
+
+### Enhancements
+
+* Update bugsnag-cocoa to v6.16.5
+* Update bugsnag-android to v5.22.0
```

Comment:
Usually we make native bumps a patch release only, so let's go with 2.0.1.

Also, it would be good to indicate the delta of the bump (especially for Cocos as updates are less frequent). Can you us the formatting that we recently used for Unreal [here](https://github.com/bugsnag/bugsnag-unreal/blob/next/CHANGELOG.md?plain=1#L15-L16)?